torch/csrc/jit/jit_opt_bisect.cpp

```diff
@@ -0,0 +1,121 @@
+#include <cstdlib>
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <ATen/core/function.h>
+#include <c10/util/Exception.h>
+#include <c10/util/StringUtil.h>
+#include <torch/csrc/jit/api/function_impl.h>
+#include <torch/csrc/jit/frontend/error_report.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/jit_opt_bisect.h>
+#include <torch/csrc/jit/serialization/python_print.h>
+
+namespace torch {
+namespace jit {
+
+static int parseOptLimit(const std::string& opt_limit) {
+  // TODO bad code
+  try {
+    int64_t n = std::stoi(opt_limit);
+    return n;
+  } catch (...) {
+    return -1;
+  }
+}
+
+static std::unordered_map<std::string, int64_t> parseJITBisectOption(
+    const char* option) {
+  std::stringstream in_ss;
+  in_ss << "function:";
```

Comment:
What is this "function:"? I didn't see it mentioned in the comment.

Reply:
oh sorry this was from jit_log.cpp and i forgot to delete